docs/reference/migration/migrate_7_0/mappings.asciidoc

```diff
@@ -31,6 +31,20 @@ of `_id`.
 The `_default_` mapping has been deprecated in 6.0 and is now no longer allowed
 in 7.0. Trying to configure a `_default_` mapping on 7.x indices will result in
 an error.
+
+If an index template contains a `_default_` mapping, it will fail to create new
+indices. To resolve this issue, the `_default_` mapping should be removed from
+the template. Note that in 7.x, the get templates API does not show the
```

Comment:
should the `get templates API` be capitalized `Get Templates API` and maybe hyperlink to the specific docs?

Reply:
👍